src/fastga/models/weight/mass_breakdown/b_propulsion/b1_engine_weight.py

```diff
@@ -45,6 +46,10 @@ def setup(self):
         self._engine_wrapper = BundleLoader().instantiate_component(self.options["propulsion_id"])
         self._engine_wrapper.setup(self)
 
+        self.add_input(
+            "settings:weight:propulsion:k_b1", val=np.nan, desc="Engine weight tunning factor"
+        )
```

Comment:
Would it be possible to rename this variable as `data:weight:propulsion:engine:k_factor` to match the names of the other k_factors (wing, fuselage, tails) ?

Reply:
I agree